runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -14765,14 +14765,14 @@ J9::X86::TreeEvaluator::directCallEvaluator(TR::Node *node, TR::CodeGenerator *c
       case TR::java_lang_String_andOR:
          return TR::TreeEvaluator::andORStringEvaluator(node, cg);
 
-      case TR::java_lang_String_toUpperHWOptimizedUTF16:
-         return TR::TreeEvaluator::decompressedStringToUpperCaseEvaluator(node, cg);
-      case TR::java_lang_String_toUpperHWOptimizedLatin1:
-         return TR::TreeEvaluator::compressedStringToUpperCaseEvaluator(node, cg);
-      case TR::java_lang_String_toLowerHWOptimizedUTF16:
-         return TR::TreeEvaluator::decompressedStringToLowerCaseEvaluator(node, cg);
-      case TR::java_lang_String_toLowerHWOptimizedLatin1:
-         return TR::TreeEvaluator::compressedStringToLowerCaseEvaluator(node, cg);
+      case TR::com_ibm_jit_JITHelpers_toUpperIntrinsicUTF16:
+         return TR::TreeEvaluator::UTF16StringToUpperCaseEvaluator(node, cg);
```

Comment:
We should probably rename these to `toUpperIntrinsicUTF16Evaluator` for easier search ability. And of course use the same names on Z if needed.

Reply:
Would the name `inlineToUpper` be appropriate or should I change it to something like `toUpperIntrinsic`?

Reply:
Let's go with `toUpperIntrinsic`. By grepping for the Java method we should be able to find all related methods/functions in the VM/JIT as well. This is the intention.

Reply:
Renamed functions in x and z codegen in 5fd75e8.